cmd/create/operatorroles/cmd.go

```diff
@@ -228,6 +234,18 @@ func run(cmd *cobra.Command, argv []string) {
 			ocm.Response:  ocm.Success,
 		})
 	case aws.ModeManual:
+		err = aws.GeneratePolicyFiles(r.Reporter, env, false, true, policies, credRequests)
+		if err != nil {
+			r.Reporter.Errorf("There was an error generating the policy files: %s", err)
+			r.OCMClient.LogEvent("ROSACreateOperatorRolesModeManual", map[string]string{
+				ocm.Response: ocm.Failure,
+			})
+			os.Exit(1)
+		}
+		if r.Reporter.IsTerminal() {
+			r.Reporter.Infof("All policy files saved to the current directory")
+			r.Reporter.Infof("Run the following commands to create the operator roles and policies:\n")
+		}
 		commands, err := buildCommands(r, prefix, permissionsBoundary, cluster, r.Creator.AccountID,
```

Comment:
The `buildCommands` here might need to be modified, as it assumes that policies already exist. You'll need to add commands to create those policies in the first place.